### PR TITLE
Adding Verlet Integration in Swift

### PR DESCRIPTION
**Pull request by CDsigma**
Added Verlet Integration in Swift

### CODE REVIEW

**Comment by Butt4cak3** [comment]
The code looks good, but take a look at #257. We decided to move the `print` statements from inside the verlet functions to `main` and make the functions return times and velocities instead.

**Comment by CDsigma** [comment]
Sounds good just updated my code accordingly 👍

File: contents/verlet_integration/code/swift/verlet.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        paramPos = paramPos*2 - prev_pos + acc * dt * dt
+        prev_pos = temp_pos
+    }
+    
+    return time
+}
+
+func stormerVerlet(pos: Double, acc: Double, dt: Double) -> (time: Double, vel: Double) {
+    var temp_pos, time, vel: Double
+    var paramPos = pos
```
Comment (Butt4cak3) [inline contents/verlet_integration/code/swift/verlet.swift]: Just a quick question: Why is this variable needed? Are parameters immutable by default?

File: contents/verlet_integration/code/swift/verlet.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
+func verlet(pos: Double, acc: Double, dt: Double) -> Double {
+    var temp_pos, time: Double
+    var paramPos = pos
+    var prev_pos = pos
+    time = 0.0
+    
+    while (paramPos > 0){
```
Comment (Butt4cak3) [inline contents/verlet_integration/code/swift/verlet.swift]: Ew! No space between `)` and `{`...

File: contents/verlet_integration/code/swift/verlet.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    return (time:time, vel:vel)
+}
+
+func velocityVerlet(pos: Double, acc: Double, dt: Double) -> (time: Double, vel: Double) {
+    var time, vel : Double
+    var paramPos = pos
+    vel = 0
+    time = 0
+    
+    while (paramPos > 0){
```
Comment (Butt4cak3) [inline contents/verlet_integration/code/swift/verlet.swift]: Same thing as further up. No space.

File: contents/verlet_integration/code/swift/verlet.swift (hunk 1 above)
Comment (CDsigma) [reply contents/verlet_integration/code/swift/verlet.swift]: Ya parameters in Swift are immutable by default. Your comment made me realize though that there is a much cleaner way to do the same thing. So I replaced `var paramPos = pos` with `var pos = pos`.

**Comment by CDsigma** [comment]
Just made the necessary revisions

**Comment by jiegillet** [comment]
@Butt4cak3, are you good with the changes?

**Comment by Butt4cak3** [comment]
I'm sorry that it took me so long to get back to this. I was taking a break and thought I was done with everything I was assigned to. I'll merge this as soon as I get home today.

**Comment by Butt4cak3** [comment]
Alright. I had to get into this again because it's been a while. Everything checks out, so this can go straight to master. Sorry again, @CDsigma.

**Pull request merged**